src/openforms/utils/files.py

```diff
@@ -0,0 +1,127 @@
+"""
+Implement :class:`django.db.models.FileField` related utilities.
+
+These utilities apply to file fields and subclasses thereof.
+"""
+import logging
+from typing import List, Literal
+
+from django.db import models
+from django.db.models.base import ModelBase
+from django.db.models.fields.files import FieldFile
+
+logger = logging.getLogger(__name__)
+
+LogLevel = Literal[
+    "debug",
+    "info",
+    "warning",
+    "error",
+    "exception",
+    "critical",
+    "fatal",
+]
```

Comment:
Bit overenginered for just that very specific `log_failed_deleted` function?

If anything, I want the level depend on the LOGGER settings in Django.

Reply:
this is a typehint expressing the levels, it still complies to the `LOGGING` configuration (will use the `openforms` logger). But I will hardcode it to `warning` indeed.